consul/plan.sh

```diff
@@ -0,0 +1,28 @@
+pkg_origin=core
+pkg_name=consul
+pkg_version=0.7.0
+pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
+pkg_license=("Apache 2")
+pkg_upstream=https://www.consul.io/
+pkg_source=https://releases.hashicorp.com/${pkg_name}/${pkg_version}/${pkg_name}_${pkg_version}_linux_amd64.zip
+pkg_shasum=b350591af10d7d23514ebaa0565638539900cdb3aaa048f077217c4c46653dd8
+pkg_filename=${pkg_name}-${pkg_version}_linux_amd64.zip
+pkg_deps=()
+pkg_build_deps=(core/unzip)
+pkg_bin_dirs=(bin)
+pkg_include_dirs=(include)
+pkg_expose=(8400 8500 8600)
+pkg_svc_user=root
+
+do_unpack() {
+  cd "${HAB_CACHE_SRC_PATH}" || exit
+  unzip ${pkg_filename} -d "${pkg_name}-${pkg_version}"
+}
+
+do_build() {
+    return 0
+}
+
+do_install() {
+  install -D consul "${pkg_prefix}"/bin/consul
```

Comment:
Simplify the unpack/install. Remove the custom `do_unpack` and use the inflated binary directly:
```bash
install -D "${HAB_CACHE_SRC_PATH}/consul" "${pkg_prefix}"/bin/consul
```